composer.json

```
@@ -15,27 +15,36 @@
     }
   },
   "require": {
-    "php": ">=7.1",
-    "doctrine/orm": "^2.6"
+    "php": "^7.4 || >=8.1",
+    "doctrine/orm": "^2.14.3"
   },
   "require-dev": {
-    "roave/security-advisories": "dev-master",
-    "phpstan/phpstan": "^0.10.0",
-    "phpstan/phpstan-strict-rules": "^0.10.0",
+    "roave/security-advisories": "dev-latest",
     "mockery/mockery": "^1.0",
-    "marc-mabe/php-enum": "^3.0",
-    "consistence/consistence": "^1.0",
+    "marc-mabe/php-enum": "^3.2 || ^4.4",
     "myclabs/php-enum": "^1.6",
     "nette/neon": "^2.4",
-    "nette/tester": "^2.0"
+    "nette/tester": "2.4.3",
+    "brandembassy/coding-standard": "10.0.0-beta",
+    "symfony/cache": "^6.2",
```

To fix the GAs? You need that for the doctrine/annotations, right?
```suggestion
    "symfony/cache": "^5.4 || ^6.2",
```

---

Its required by \Doctrine\ORM\Tools\Setup::createCacheInstance, I think its more general requirement than just the annotations though.